OData/src/System.Web.OData/OData/Routing/ODataPathSegmentTranslator.cs

```diff
@@ -18,7 +18,7 @@ namespace System.Web.OData.Routing
     /// <summary>
     /// Translator to convert an OData path segment to WebAPI path segment.
     /// </summary>
-    internal class ODataPathSegmentTranslator : PathSegmentTranslator<IEnumerable<ODataPathSegment>>
+    public class ODataPathSegmentTranslator : PathSegmentTranslator<IEnumerable<ODataPathSegment>>
```

Comment:
Change TranslateODLPathToWebAPIPath to such as: TranslateODataLibPathToWebApiPath, or add to a dictionary https://github.com/OData/WebApi/blob/master/OData/src/CodeAnalysisDictionary.xml